mongoDB.go

```diff
@@ -113,6 +117,47 @@ func mapUniqueRecords(records *[]RecordLink) map[string]aggregatedRecord {
 	return recordMap
 }
 
+// Attempts to read the geometry from the Postgres table and update to Mongo
+func findElementPoints(s *Server, session *mgo.Session, record *RecordLink, tableMap map[int]TableMeta) {
+	// Only update if no lat or lon information stored.
+	if record.Lat < 0.1 && record.Lat > -0.1 && record.Lon < 0.1 && record.Lon > -0.1 {
```

Comment:
Yes thought it might be floating point problems. This is the way to do it then.

It's a pity there is no other way of checking (null value, another flag)